### DISCUSSION TITLE
Zotero Feature

### DISCUSSION THREAD

**Discussion by MaPeras** [knowledgefutures/pubpub#2609] [General]
Hello,

foremost – that's spectacular! Currently, I can't find the feature at openrewi.pubpub.org – are you going to roll out this feature to every community? Looking forward to test this.

Zotero integration would be even better, if we could add custom citation styles. Those can differ widely in the non-English standardized world. 

Cheers
Max

**Comment by gabestein** [comment]
Hi Max — apologies, the email got ahead of the rollout. It's now live!
